streams/src/main/java/org/apache/kafka/streams/processor/assignment/assignors/StickyTaskAssignor.java

```diff
@@ -105,6 +105,8 @@ private void optimizeActive(final ApplicationState applicationState,
             .filter(task -> !task.isStateful())
             .map(TaskInfo::id)
             .collect(Collectors.toSet());
+
+        // TODO: We need to use stateless traffic costs and non overlap costs here.
```

Comment:
You should probably take this out just because you're already working on the PR that addresses this, and would have to rebase it to remove the TODO comment when  this PR is merged which means retriggering the build on your other PR in like 4 hours